### PR TITLE
Users/anksing/release 1.5.0

### PR DESCRIPTION
**Pull request by singankit**
# Description

# Checklist

- [X] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [X] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/python/assets/assets-in-registry/share-models-components-environments.ipynb [hunk 1]
```diff
@@ -583,9 +586,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3.10 - SDK V2",
```
Comment (harneetvirk) [inline sdk/python/assets/assets-in-registry/share-models-components-environments.ipynb:586]: Why do we need to change kernel spec?

Comment (singankit) [reply sdk/python/assets/assets-in-registry/share-models-components-environments.ipynb]: Reverted this change

File: sdk/python/assets/assets-in-registry/share-models-components-environments.ipynb [hunk 2]
```diff
@@ -598,11 +601,6 @@
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
    "version": "3.8.8"
-  },
-  "vscode": {
-   "interpreter": {
```
Comment (harneetvirk) [inline sdk/python/assets/assets-in-registry/share-models-components-environments.ipynb:603]: Is this delete expected?

Comment (singankit) [reply sdk/python/assets/assets-in-registry/share-models-components-environments.ipynb:603]: The tests passed with when this change came in so guessing should be okay.

**Review by harneetvirk — APPROVED** [review]

**Pull request merged**